### DISCUSSION TITLE
I want Galaxy Brain badges, but how can I complete 2 Accepted Answers?

### DISCUSSION THREAD

**Discussion by fynntang** [community/community#18796] [Other Feature Feedback, Questions, & Ideas] [answered]
I want Galaxy Brain badges, but how can I complete 2 Accepted Answers?

**Comment by thomscoder** [comment]
By answering other people questions in discussions and having them accepting your answers :)
You need 2 to earn the badge.
More on: https://nine-five.ghost.io/github-achievements/
Repo: https://github.com/Schweinepriester/github-profile-achievements
Official blog: https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

**Reply by kartikxisk** [reply]
thanks

**Reply by RoshdiRaed** [reply]
great

**Reply by IgorMariano25** [reply]
That is nice !

**Reply by shox404** [reply]
Thanks bro .

**Comment by melvincwng** [comment]
@fynntang, to get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

1. Discussions should be in a public repository
2. Get marked as answers to 2 or more questions
3. The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
4. In fact, answering your own questions could prevent the achievement from displaying in your profile page.
5. My advice here is to perhaps scour a few discussion forums, and answer a few questions based on your own knowledge/expertise. Perhaps if someone finds your answer helpful, he/she would mark your answer and thus help you to unlock the badge.

Hope these steps help you in achieving your badge! Good luck :)

**Reply by Guedesou** [reply]
Thanks, Melvin! Helped us a looot!

**Reply by chiragmehta900** [reply]
Thanks, Melvincwng!  it's helped a lot.

**Reply by cybertcode** [reply]
Genial muchas gracias

**Reply by kartikxisk** [reply]
thanka [melvincwng](https://github.com/melvincwng)

**Reply by thaissevero** [reply]
Okay guys that was great, I want it too! hahaha

**Reply by HEEPOKE** [reply]
Thanks guys

**Reply by XJ4v1erX** [reply]
thanks for the help :)

**Reply by IgorMariano25** [reply]
thanks for the help :)

**Reply by itsdotnet** [reply]
Thanks guys

**Comment by alamin-karno** [comment]
After answering two discussions, You will get a Galaxy Brain badge.

Note that: Discussions must be on a public repository & the answer must be marked as answered by whom the question is asked. For example, you ask this question. You may mark my answer as the current answer. If I do this kind of work twice, I will get the badge.

More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Reply by SatyamV7** [reply]
> After answering two discussions, You will get a Galaxy Brain badge.
> 
> Note that: Discussions must be on a public repository & the answer must be marked as answered by whom the question is asked. For example, you ask this question. You may mark my answer as the current answer. If I do this kind of work twice, I will get the badge.
> 
> More details here: https://github.com/Schweinepriester/github-profile-achievements

Yes, This is correct

**Reply by passiondriven** [reply]
Yes . I think so!

**Comment by alanny-design** [comment]
2 accepted answers for galaxy brain badge

**Comment by Andreaa-Dev** [comment]
You need to got 2 accepted answers to earn the  galaxy brain badge.

**Reply by passiondriven** [reply]
Thank for your explanation!

**Comment by nemmtor** [comment]
Your 2 answers have to get accepted 🙂 good luck 🎉

**Comment by fsadykov** [comment]
GitHub has very nice documentation to be able to get **Galaxy Brain** your 2 answers should be accepted 
https://github.com/Schweinepriester/github-profile-achievements

**Comment by strikeenzo** [comment]
You must get 2 accepted answers to a public repository. You cannot answer your own questions, this will not work and/or count. Answers must be accepted by another member.

**Comment by Pathum-lakshan** [comment]
@fynntang,


to get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

Discussions should be in a public repository
Get marked as answers to 2 or more questions
The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
In fact, answering your own questions could prevent the achievement from displaying in your profile page.
My advice here is to perhaps scour a few discussion forums, and answer a few questions based on your own knowledge/expertise. Perhaps if someone finds your answer helpful, he/she would mark your answer and thus help you to unlock the badge.
Hope these steps help you in achieving your badge! Good luck :)

**Comment by Veercodeprog** [comment]
to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

**Comment by IgorMariano25** [comment]
@Veercodeprog Could you help me get the Galaxy Brain ? 🥹

**Comment by YasinDehfuli** [comment]
If you need step by step, how to get galaxy brain achievement by image, you can use this respository how to get galaxy brain link:

https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.galaxy-brain.md

**Comment by github-staff** [comment, marked as answer]
Thanks so much for posting in our Community @fynntang. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Andrewrafat** [comment]
can i git it too ?

**Comment by miketopeak** [comment]
https://github.com/sagelga/github-achievement
you can check this url for your purpose. 

How to get: Answers get accepted {this.amount} in Discussion.

2 Accepted Answers in Github Discussions
also available in Bronze, Silver, and Gold achievements.

8 Accepted Answers in GitHub Discussions
16 Accepted Answers in GitHub Discussions
32 Accepted Answers in GitHub Discussions

**Reply by ebndev** [reply]
Hi @miketopeak,

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).
